.travis.yml

```diff
@@ -13,16 +13,15 @@ matrix:
   allow_failures:
     - d: dmd-nightly
 
-env:
-  - DUB_CONFIGURATION=python27
-
 install:
   - virtualenv autowrapenv
   - source autowrapenv/bin/activate
   - pip install pytest
 
 script:
-  - make
+  - DUB_CONFIGURATION=python27 make
+  - dub run pyd:setup && source pyd_set_env_vars.sh
```

Comment:
Why is this needed? Especially with the version being selected as 2.7?

Reply:
you have to `source pyd_set_env_var.sh` in order to use the `env` config.

Reply:
I need to look at `pyd_set_env_var.sh` to see what it's doing. There has to be a better way. As a user, I should activate a virtualenv and things should just work. No sourcing, no shell scripts, no nothing.

Reply:
I have thought about this a lot over several years. Obviously I may have missed something but as far as I can tell this is the only solution that cuts it.